merlin.spec

```diff
@@ -278,8 +283,8 @@ systemctl enable merlind.service
 # If mysql-server is running _or_ this is an upgrade
 # we import logs
 if [ $1 -eq 2 ]; then
-  mon log import --incremental || :
-  mon log import --only-notifications --incremental || :
+  mon log import --incremental > /dev/null 2>&1 || :
+  mon log import --only-notifications --incremental > /dev/null 2>&1 || :
```

Comment:
Would it be enough to discard `stdout`? I don't know how `mon log report` works in this regard, but it might be desireable to still see any errors in the log, if any? Either way, this should be an improvement.